mlflow/store/tracking/file_store.py

```diff
@@ -507,7 +507,7 @@ def _get_deleted_runs(self, older_than=0):
         for deleted_run in deleted_runs:
             _, run_dir = self._find_run_root(deleted_run.info.run_uuid)
             meta = read_yaml(run_dir, FileStore.META_DATA_FILE_NAME)
-            if "deleted_time" not in meta or current_time - int(meta["deleted_time"]) > older_than:
+            if "deleted_time" not in meta or current_time - int(meta["deleted_time"]) >= older_than:
```

Comment:
If this method (`_get_deleted_runs`) is called right after the delete-run operation within 1 millisecond, `current_time - int(meta["deleted_time"]` becomes 0 so `=` is needed.